ui/apps/platform/src/Containers/NetworkGraph/components/StyleFakeGroup.tsx

```diff
@@ -11,6 +11,7 @@ import {
 import AlternateIcon from '@patternfly/react-icons/dist/esm/icons/regions-icon';
 import DefaultIcon from '@patternfly/react-icons/dist/esm/icons/builder-image-icon';
 import useDetailsLevel from '@patternfly/react-topology/dist/esm/hooks/useDetailsLevel';
+import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
```

Comment:
Can you search PatternFly repo to see if there is an issue for next major version 5 to improve like:

```ts
import { SVGIconProps } from '@patternfly/react-icons';
```

Reply:
looks like there aren't :/